auth/keycloak/config/realm-export.json

```diff
@@ -1102,6 +1102,22 @@
       "authenticationFlowBindingOverrides": {},
       "fullScopeAllowed": true,
       "nodeReRegistrationTimeout": -1,
+      "protocolMappers": [
```

Comment:
This allows keycloak to return roles(groups) into the userinfo object by default. This means that the keycloak library that we are using throughout the application has this information in context.

Reply:
Might need to do this in DEV.  I'd also be careful with linking Groups to front-end functionality.  The groups are designed to be arbitrary.  Functionality is only concerned with 'Keycloak:Roles/PIMS:Claims'.

Reply:
Don't need to change this PR.